Code/Scripts/gameobjobserver.h

```diff
@@ -42,6 +42,7 @@
 #endif
 
 #include <stdlib.h>
+#include <stdint.h>
```

Comment:
This one is the only required include.

All other files include `gameobjobserver.h` so don't need their own `#include <stdint.h>`.
`uintptr_t` is needed for the purely virtual `GameObjObserverClass::Custom`.

Reply:
Fair enough. At some point we'll hook up include-what-you-use to clean up unnecessary includes.